crates/wasmparser/src/readers/core/tags.rs

```diff
@@ -24,6 +24,12 @@ impl<'a> FromReader<'a> for TagType {
         if attribute != 0 {
             bail!(reader.original_position() - 1, "invalid tag attributes");
         }
+        if !reader.exceptions() {
+            bail!(
+                reader.original_position(),
+                "exceptions proposal not enabled"
+            );
+        }
```

Comment:
Mind moving this back to the validator? (and tests we have to `assert_invalid`)

Reply:
Done.